README.md

```diff
@@ -86,6 +86,7 @@ when a real user uses it.
   - [`getByAltText`](#getbyalttext)
   - [`getByTitle`](#getbytitle)
   - [`getByValue`](#getbyvalue)
```

Comment:
Let's remove the docs for `getByValue` and `getBySelectText` because we want to encourage people to use the `getByDisplayValue` instead.